docs/extend/plugins_volume.md

```diff
@@ -164,3 +164,52 @@ this point.
 
 Respond with a string error if an error occurred.
 
+
+### /VolumeDriver.Get
+
+**Request**:
+```
+{
+    "Name": "volume_name"
+}
+```
+
+Get the volume info.
+
+
+**Response**:
+```
+{
+  "Name": "volume_name",
+  "Mountpoint": "/path/to/directory/on/host",
+  Err: ""
+}
+```
+
+Respond with a string error if an error occurred.
+
+
+### /VolumeDriver.List
+
+**Request**:
+```
+{}
+```
+
+Get the list of volumes registered with the plugin.
+
+**Response**:
+```
+{
+  Volumes: [
+    {
+      "Name": "volume_name",
+      "Mountpoint": "/path/to/directory/on/host"
+    }
+  ],
+  Err: ""
+}
```

Comment:
@cpuguy83 perhaps you can change all "fences", and add a JSON hint, i.e.;

```
   ```JSON
```

(also happy to do a follow up PR to fix that in all the Plugin docs at once)